### ISSUE TITLE
Ch05 - Unleash the Power of Transfer Learning / Basic CNN model

### ISSUE THREAD

**Issue by yingwei13mei** [dipanjanS/hands-on-transfer-learning-with-python#3]
Hi,
   I used this book's code to learn in Ch05, Basic CNN Model.ipynb, after running these codes, I found my validate_acc always 50%, and it didn't change any more.
   I guess the following code may have to modify:
"
model.compile(loss='binary_crossentropy',
              optimizer=optimizers.RMSprop(),
              metrics=['accuracy'])
"
  Another optimizer could help to get a better accuracy? 
  I am a starter.

**Comment by dipanjanS** [comment]
I don't think so, while you can always experiment with optimizers in general, RMSprop is quite effective especially for fine-tuning models also. There could be multiple reasons for it not increasing ranging from data normalization to batch size to the actual data you have. We recently repeated these experiments for this article and it still worked fine: https://towardsdatascience.com/a-comprehensive-hands-on-guide-to-transfer-learning-with-real-world-applications-in-deep-learning-212bf3b2f27a

You might want to check your data distribution, what you are doing for image normalization\scaling\preprocessing and how you are training it in general. But feel free to experiment with other optimizers, architectures also.
